about_cookie.html

```diff
@@ -56,7 +56,7 @@
 
         <div class="row">
             <h1 class="text-left text-info heading--white ">Cookie policy</h1>
-            <p class="text-left text-info heading--white ">We use cookie on our website for convenient operation.</p>
+            <p class="text-left text-info heading--white "> We use cookie on our website for convenient operation. You can always turn it off. You shoud do this in browser settings.</p>
```

Comment:
I think it would be better to write:

"You can always turn it off by changing your browser settings."